### ISSUE TITLE
Add Repository Link In Home Page

### ISSUE THREAD

**Issue by Coding-Hashira** [pushkaraj2007/Threadify#1]
On the home page, it's mentioned that the project is open sourced, but the repository's link isn't given anywhere.

I believe you can add the repository link in the socials tab on the home page:
![IMG_20221009_093844.jpg](https://user-images.githubusercontent.com/96920778/194737526-d08a4668-d0f2-48f5-8327-2f984144ce2b.jpg)

**Comment by pushkaraj2007** [comment]
Do you want to work on this?

**Comment by Coding-Hashira** [comment]
Yes! 

BTW, I think that before this improvement, you should actually make other social links work.

**Comment by pushkaraj2007** [comment]
Should we create seperate social media accounts for threadify?

**Comment by Coding-Hashira** [comment]
If you're thinking to scale it, definitely you should. Plus, building in public is fun! 😉

**Comment by pushkaraj2007** [comment]
Definitely want to scale!! 

I think for now Twitter and GitHub repository is fine, we can create others in the future.

**Comment by Coding-Hashira** [comment]
Yes, just update the Twitter link, and I'll do the rest!

**Comment by pushkaraj2007** [comment]
I'll do it soon...

**Comment by prakhartiwari0** [comment]
May I work on it? @pushkaraj2007

**Comment by subashpaul** [comment]
can i work on this?

**Comment by pushkaraj2007** [comment]
@subashpaul Sorry but @prakhartiwari0 asked first, going to assign him for this project. 

@subashpaul you can work on other issues.

**Comment by prakhartiwari0** [comment]
Wooh! @pushkaraj2007 Let me be honest here, I don't know tailwind, typescript, react and next. Therefore it is going to be a little harder for me, if you want to increase the speed you may wait for somebody else more capable than me to handle this issue.
But, I am ready to learn whatever it demands me to, till now I have understood that I need to understand the tailwind CSS and maybe next and react. **I would be really grateful to you if you can guide me** on what _exactly_ I need to learn which will make me capable of at least working on this issue, and if possible, on other issues too.

What I know currently: Only HTML CSS pretty well, I know JS basics only, which is definitely not enough I guess.

I can learn things and then contribute, it will surely take time.

**Comment by prakhartiwari0** [comment]
Also, may I know from where you took the SVG codes for icons? I am able to understand a few things so trying to work with that.

**Comment by pushkaraj2007** [comment]
@prakhartiwari0 

Paste the exact code given below in components/Footer.tsx file, and create a pull request

```
import { NextPage } from "next";

const Footer: NextPage = () => {
    return (
        <footer className="text-gray-600 body-font">
            <div className="container px-5 py-8 mx-auto flex items-center sm:flex-row flex-col">
                <a className="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
                    <p className="text-[20px]">🧵</p>
                    <span className="ml-3 text-xl">Threadify</span>
                </a>
                <p className="text-sm text-gray-500 sm:ml-4 sm:pl-4 sm:border-l-2 sm:border-gray-200 sm:py-2 sm:mt-0 mt-4">© 2022 Threadify —
                    <a href="https://twitter.com/pushkaraj2007" className="text-gray-600 ml-1" rel="noopener noreferrer" target="_blank">Pushkaraj Kulkarni</a>
                </p>
                <span className="inline-flex sm:ml-auto sm:mt-0 mt-4 justify-center sm:justify-start">
                    <a href="https://github.com/Threadify" className="text-gray-500">
                        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"><path d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.23.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z" /></svg>
                    </a>
                    <a href="https://twitter.com/pushkaraj2007" className="ml-3 text-gray-500">
                        <svg fill="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" className="w-5 h-5" viewBox="0 0 24 24">
                            <path d="M23 3a10.9 10.9 0 01-3.14 1.53 4.48 4.48 0 00-7.86 3v1A10.66 10.66 0 013 4s-4 9 5 13a11.64 11.64 0 01-7 2c9 5 20 0 20-11.5a4.5 4.5 0 00-.08-.83A7.72 7.72 0 0023 3z"></path>
                        </svg>
                    </a>
                </span>
            </div>
        </footer>
    )
}

export default Footer
```

**Comment by pushkaraj2007** [comment]
@prakhartiwari0 

You will learn it along the way, happy contributing :)

**Comment by Coding-Hashira** [comment]
@prakhartiwari0 You can also try and help us with no code contributions along with learning code! 😉

**Comment by pushkaraj2007** [comment]
!! This Issue has been resolved in #5 PR !!
